starboard/linux/shared/platform_configuration/BUILD.gn

```diff
@@ -143,7 +143,7 @@ config("platform_configuration") {
     "rt",
   ]
 
-  if (!sb_is_modular) {
```

Comment:
Let's keep just !sb_is_modular

Reply:
i get this error if i just keep the conditions as if (!(sb_is_modular)) {

Error goes away if i do  if (!sb_is_modular || current_toolchain != cobalt_toolchain) {

[4464/4465] LINK starboard/storage_test_loader
FAILED: starboard/storage_test_loader 
python "../../build/toolchain/gcc_link_wrapper.py" --output="starboard/storage_test_loader" -- ../../../../starboard-toolchains/x86_64-linux-gnu-clang-chromium-16-init-17653-g39da55e8-2/bin/clang++ -Wl,-rpath=/usr/local/google/home/nyardi/cobalt_github/cobalt/out/linux-x64x11_devel_modular/starboard -Wl,-rpath=/usr/local/google/home/nyardi/cobalt_github/cobalt/out/linux-x64x11_devel_modular -Wl,-rpath=\$ORIGIN -Wl,-rpath=/usr/local/google/home/nyardi/cobalt_github/cobalt//third_party/libvpx/platforms/linux-x64/ -Wl,--wrap=eglSwapBuffers -fuse-ld=lld -fsanitize=address -Wl,-u_sanitizer_options_link_helper -Wl,-rpath=\$ORIGIN/lib -Wl,-gc-sections ../../third_party/libvpx/platforms/linux-x64/libvpx.so.6 -lX11 -lXcomposite -lXrender -lasound -ldl -lpthread -lrt -o "starboard/storage_test_loader" -Wl,--start-group @"starboard/storage_test_loader.rsp" ./libstorage_test.so starboard/libstarboard_platform_group.so -Wl,--end-group 
ld.lld: error: undefined reference due to --no-allow-shlib-undefined: net::HttpServer::~HttpServer()
>>> referenced by ./libstorage_test.so

ld.lld: error: undefined reference due to --no-allow-shlib-undefined: net::HttpServer::GetLocalAddress(net::IPEndPoint*)
>>> referenced by ./libstorage_test.so